MoppApp/MoppApp/AddresseeViewController.swift

```diff
@@ -98,14 +98,21 @@ class AddresseeViewController : MoppViewController {
 
         Task {
             let result = await OpenLdap.search(identityCode: trimmedText)
-            if (result.count > 0) {
-                self.foundAddressees = result.sorted { $0.identifier < $1.identifier }
-                self.showLoading(show: false)
-                self.tableView.reloadData()
-            } else {
-                self.infoAlert(message: "\(L(.cryptoEmptyLdapLabel))")
-                self.showLoading(show: false)
+            let filteredResults = await OpenLdap.filteredResults(addressees: result)
+
+            showLoading(show: false)
+
+            guard !result.isEmpty else {
```

Comment:
Why not return total search count first place and not waste memory for unused results?

Reply:
I was thinking do filtering still in OpenLDAP.search and return ([Addressee], Int) as found result count

Reply:
Changed